<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-SJO5CKK2" failures="1" tests="5" name="Test" time="26.5" errors="0" timestamp="2023-05-17T07:51:00 IST">
  <testcase classname="com.demo.automation.DemoTest" name="demo1TestCase" time="15.832"/>
  <testcase classname="com.demo.automation.DemoTest" name="demo2TestCase" time="0.003">
    <skipped/>
  </testcase> <!-- demo2TestCase -->
  <testcase classname="com.demo.automation.DemoTest" name="demo2TestCase" time="0.003">
    <skipped/>
  </testcase> <!-- demo2TestCase -->
  <testcase classname="com.demo.automation.DemoTest" name="demo2TestCase" time="0.002">
    <skipped/>
  </testcase> <!-- demo2TestCase -->
  <testcase classname="com.demo.automation.DemoTest" name="demo2TestCase" time="0.002">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [true] but found [false]
at com.demo.automation.DemoTest.demo2TestCase(DemoTest.java:38)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- demo2TestCase -->
</testsuite> <!-- Test -->
